docs/controllers/generics.md

```diff
@@ -0,0 +1,113 @@
+# Generics
+
+This chapter contains tips and tricks for re-using generic reconcilers.
+
+## Generic Reconcilers
+It is possible to create generic reconcilers by abstracing away the underlying type by using either [DynamicObject] or [PartialObjectMeta].
```

Comment:
why are they enclosed in square bracket? is that a convention or do you want to add urls?

Reply:
after reading the full doc I guess that when it is finally rendered in doc.rs or kube.rs the ones in square bracket would get linked to the traits or obj def. is that correct?

Reply:
yes, this is wiki-style links from the links file.